CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+## [day-two 0.1.0-alpha.2](https://github.com/talos-systems/day-two/releases/tag/v0.1.0-alpha.2) (2022-04-08)
+
+Welcome to the v0.1.0-alpha.2 release of day-two!
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/day-two/issues.
+
+### Contributors
+
+* Spencer Smith
+* Tim Jones
+* Tim Jones
```

Comment:
Still need to make this list unique contributors!

Reply:
you changed your email ;) there's a way to merge contributors by specifying email aliases, but this is manual